_docs/1.0.0/usage-errcol.md

```diff
@@ -10,3 +10,10 @@ Error Column
 ============
 
 `errCol` description will come here
+
+### Random notes
```

Comment:
I don't like random notes in the GH Pages in all honesty. But if others are cool with it I won't protest.

Reply:
I understand.
The choice here was to create this stub or risk the loss of the information (I already had forgotten about it, just found the note in my to-do list.) . I'll keep in mind to put this into better shape soon. 😉

Reply:
maybe "Other notes"?